tests/unit-tests/UnitTests/Validators/IntegrityChecks/Form/TemplatedEmailActionCheckTests.cs

```diff
@@ -12,6 +12,7 @@ public class TemplatedEmailActionCheckTests
         [InlineData("", "templateId", "emailAddress", "FAILURE - Templated Email Action, there is no 'EmailTemplateProvider'")]
         [InlineData("provider", "", "emailAddress", "FAILURE - Templated Email Action, there is no 'TemplateId' provided")]
         [InlineData("provider", "templateId", "", "FAILURE - Templated Email Action, there is no 'To' provided")]
+        [InlineData("provider", "templateId", "emailAddress", "FAILURE - Templated Email Action, there is no 'Personlisation' field(s) added")]
         public void TemplatedEmailActionCheck_ShouldAddCorrectErrorMessage_WhenActionIsMissingProperties(
             string provider, string templateId, string to, string expectedErrorMessage)
         {
```

Comment:
Can personalisation be added to the parameters for the inline data so that each test only has one failure? Currently all the tests will fail on personalisation being missing